### PR TITLE
Adjust staking logic to match updated contract

### PR DESCRIPTION
**Pull request by Adamantios**

### CODE REVIEW

File: packages/valory/skills/staking_abci/rounds.py [hunk 1]
```diff
@@ -48,9 +48,18 @@ class Event(Enum):
     ROUND_TIMEOUT = "round_timeout"
     NO_MAJORITY = "no_majority"
     SERVICE_NOT_STAKED = "service_not_staked"
+    SERVICE_EVICTED = "service_evicted"
     NEXT_CHECKPOINT_NOT_REACHED_YET = "next_checkpoint_not_reached_yet"
 
 
+class StakingState(Enum):
```
Comment (jmoreira-valory) [inline packages/valory/skills/staking_abci/rounds.py:55]: Why is this not part of the contract?

**Review by jmoreira-valory — APPROVED** [review]

File: packages/valory/skills/staking_abci/rounds.py (hunk 1 above)
Comment (Adamantios) [reply packages/valory/skills/staking_abci/rounds.py]: 93b7dc8f10b67df1e30ed607048627d82e73acaf

File: packages/valory/contracts/service_staking_token/contract.py [hunk 2]
```diff
@@ -19,28 +19,38 @@
...
         ledger_api: LedgerApi,
         contract_address: str,
         service_id: int,
     ) -> JSONLike:
         """Check whether the service is staked."""
         contract_instance = cls.get_instance(ledger_api, contract_address)
-        res = contract_instance.functions.isServiceStaked(service_id).call()
-        return dict(data=res)
+        res = contract_instance.functions.getServiceStakingState(service_id).call()
+        return dict(data=StakingState(res))
```
Comment (jmoreira-valory) [inline packages/valory/contracts/service_staking_token/contract.py:53]: This is not returning the correct value. It returns StakingState.UNSTAKED for service ID 275, which you can verify it is staked here:
https://gnosisscan.io/address/0xf8a415d761b1279B0f1ACE68cD85e5ca4346Cc3F#readContract

Comment (Adamantios) [reply packages/valory/contracts/service_staking_token/contract.py:53]: be6749a35163b8c4aa752a0c003e19eb3006b29f

**Review by angrybayblade — APPROVED** [review]

**Pull request merged**